# Домашнее задание первого урока



**Задание 1**

**Два человека одновременно стреляют по мишени. Вероятность попадания по мишени у первого стрелка равна 0,5, у второго — 0,8. Какова вероятность того, что в мишени будет только одна пробоина?**

Примечание: в этой и последующих задачах десятичным разделителем должна быть точка. 

0.5 * (1 - 0.8) + 0.8 * (1 - 0.5) = `0.5`

**Задание 2**

**Трое аналитиков на экзамене независимо друг от друга решают одну и ту же задачу. Вероятности ее решения этими аналитиками равны 0,8, 0,7 и 0,6 соответственно. Найдите вероятность того, что хотя бы один аналитик решит задачу.**

Это событие обратное тому, что никто не решит задачу:

1 - (1 - 0.8) * (1 - 0.7) * (1 - 0.6) = `0.976`


**Задание 3**

**В первой урне находятся 10 белых и 4 черных шаров, а во второй 5 белых и 9 черных шаров. Из каждой урны вынули по шару. Какова вероятность того, что оба шара окажутся черными? В ответ запишите значение, округленное до тысячных.**

4/14 * 9/14 = `0.18367346938775508`


**Задание 4**

**Вероятность хотя бы одного попадания в цель при четырех выстрелах равна 0,9984. Найдите вероятность попадания в цель при одном выстреле.**

( p ) — вероятность попадания в цель при одном выстреле.
( 1 - p ) — вероятность промаха при одном выстреле
[ (1 - p)^4 ] вероятность промаха во всех 4 выстрелах
[ 1 - (1 - p)^4 = 0,9984 ] вероятность хотя бы одного попадания
 ли 1 - (1 - p)^4 = 0.9984

(1 - p)^4 = 1 - 0.9984

(1 - p)^4 = 0.0016  #потом извлекаем корень четвертой степени

1 - p = 0.2

-p = -0.8

p = `0.8`


**Задание 5**

**Из колоды в 36 карт наудачу извлекается одна карта. События:**

A = {вынутая карта – туз}

В = {вынутая карта чёрной масти}

F = {вынутая карта – фигура, т.е. является валетом, дамой, королём или тузом}.

**Выберите все пары зависимых событий:**

* А и В
* **A и F**
* B и F

**Задание 6**

**В конверте находится 10 лотерейных билетов, среди которых 3 выигрышных. Из конверта последовательно извлекаются билеты. Найти вероятность того, что три первых лотерейных билета будут выигрышными. Ответ округлите до десятитысячных.**

In [3]:
round(3/10*2/9*1/8, 4)

0.0083

Ответ: `0.0083`

**Задание 7**

**В трех углах равностороннего треугольника находится по роботу. Каждый из роботов начинает двигаться в случайно выбранный угол по стороне треугольника. Какова вероятность того, что ни один из роботов не столкнется с другим роботом?**

Ответ: каждый робот может пойти в две стороны, всего роботов три. Значит, число возможных комбинаций направлений таково: 2 × 2 × 2 = 8. Из всех комбинаций только две удовлетворяют условию, что они не встретятся.

Вспоминаем формулу вычисления вероятностей: p = m ÷ n, где m — число исходов, благоприятствующих осуществлению события, а n — число всех равновозможных исходов. Подставим наши цифры: `2 ÷ 8 = ¼`. Значит, шанс избежать столкновения — `один из четырёх`.


**Задание 8**

**Брошены две игральные кости. Есть следующие события:**

А = {выпадение шестерки на первой кости}

В = {сумма выпавших очков равна 7}

С = {выпадение тройки на второй кости}

**Какие пары событий являются независимыми?**

* **В и С**
* **А и С**
* **А и В**


**Задание 9**

**Бросаются 2 игральных кубика. Определите вероятность того, что произведение числа очков на обоих кубиках не превосходит 10.**

Ответ: благоприятные варианты (1,1), (1,2), (1,3), (1,4), (1,5), (1,6), (2,1), (2,2), (2,3), (2,4), (2,5), (3,1), (3,2),(3,3),(4,1), (4,2),(5,1), (5,2), (6,1) Итого 19 вариантов `Р=19/36`, т.е. `0.528`.